nemo_skills/pipeline/utils/mounts.py

```diff
@@ -118,6 +137,20 @@ def check_mounts(
         # Just check if the paths are mounted
         for path in mount_map.keys():
             if path is not None:
+                if "${" in path and "}" in path:
+                    if env_vars is None:
+                        env_vars = get_env_variables(cluster_config)
+
+                    # Replace ${VAR} with the value from env_vars
+                    for var in env_vars:
+                        if var not in env_vars:
+                            raise ValueError(
+                                f"Cluster config item `{path}` contains env variable placeholders, but the required env var is "
+                                f"not provided in the cluster config or environment to resolve."
+                            )
+
+                        path = path.replace(f"${{{var}}}", env_vars[var])
```

Comment:
Same here. Was `os.path.expandvars` not sufficient?

Reply:
Updated